contracts/src/v0.8/dev/interfaces/OCR2DROracleInterface.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR oracle interface.
+ */
+interface OCR2DROracleInterface {
+  function sendRequest(
+    uint256 nonce,
+    uint256 subscriptionId,
```

Comment:
Shouldn't subscriptionID be derived from caller address? If not, then we need to validate it so people won't cheat and try to use somebody else's ID.

Reply:
The billing system must verify the origin address and must not rely on subscriptionID. Another reason is that a user can use the same subscription for multiple user contracts.

Reply:
Got it. Please put this into a comment.

Reply:
Done